packages/hooks-react/src/useBootstrapApp.ts

```diff
@@ -16,18 +14,17 @@ type Resources = {
   settings: Settings;
 };
 
-export const useBootstrapApp = (onReady: () => void) => {
-  const { data, isLoading, error, isSuccess, refetch } = useQuery<Resources, AppError>('config-init', applicationController.initializeApp, {
+export type OnReadyCallback = (config: Config | undefined) => void;
+
+export const useBootstrapApp = (url: string, onReady: OnReadyCallback) => {
+  const { data, isLoading, error, isSuccess, refetch } = useQuery<Resources, AppError>('config-init', () => applicationController.initializeApp(url), {
```

Comment:
Instead of passing the `url` around, perhaps we can extract the `configKey` from the url params from the platform (`web/src/components/Root` for now) and send that to `useBootstrapApp` -> `initializeApp` -> `getConfigSource`.

Reply:
The `url` param is also used by the integration initialization. But the AppController doesn't know what the integration wants to know, so I just pass the full URL and let the integration figure it out 😄

Reply:
I see. But I find it a bit unclear what `url` contains within the bootstrap concept. Maybe another time. :)